setup.py

```diff
@@ -0,0 +1,22 @@
+import setuptools
+print('# open("requirements.txt").readlines()', open("requirements.txt").readlines())
```

Comment:
Is this really needed in setup.py?

Reply:
It was just the first draft and not needed.